Improve documentation. Added general capabilities we would like to ac… (#7)
stratisg/Cellular-automata: README.md

Work on cellular automata

## Overview
This project aims to use Python to construct cellular automata in 1D and 2D.
We want to study the importance of the different rules for the time evolution
of a cellular automaton. This package allows the user to do the following:
- Choose the system's dimensionality.
- Choose the neighborhood for each site that will determines that site's 
future state.
- Choose the function that determines the future state of a site given its
past and present states as well as those of the sites in its neighborhood.
- Build a probabilistic cellular automaton.

> - Build a cellular automaton that its governing laws are dynamic.

Reviewer: nitpick:
```suggestion
- Build a cellular automaton that whose laws are dynamic.
```
